src/apps/explorer/layout/Header.tsx

```diff
@@ -15,20 +31,26 @@ export const Header: React.FC = () => {
 
   return (
     <GenericHeader logoAlt="CoW Protocol Explorer" linkTo={`/${prefixNetwork || ''}`}>
-      <Navigation>
-        <NetworkSelector networkId={networkId} />
-        {/*      
-        <li>
-          <Link to="/">Batches</Link>
-        </li>
-        <li>
-          <Link to="/trades">Trades</Link>
-        </li>
-        <li>
-          <Link to="/markets">Markets</Link>
-        </li>
-        */}
-      </Navigation>
+      <NetworkSelector networkId={networkId} />
+      <FlexWrap ref={ref} grow={1}>
+        <MenuBarToggle isActive={isBarActive} onClick={(): void => setBarActive(!isBarActive)}>
+          <FontAwesomeIcon icon={faBars} />
+        </MenuBarToggle>
+        <Navigation isActive={isBarActive}>
+          <li>
+            <Link to="/">Home</Link>
+          </li>
+          <li>
+            <Link to="cow.fi">CoW Protocol</Link>
```

Comment:
Links aren't working. We should add:
- Complete url (e.g: https://someurl)
- External links should have an icon to indicate that they will be opened in a new tab.

Reply:
uh, i missed that part. fixed